### ISSUE TITLE
New Rule Proposal: Disallow multiple empty lines

### ISSUE THREAD

**Issue by another-dave** [sasstools/sass-lint#1015]
Hello,

I'd like to propose a rule to disallow multiple consecutive empty lines in a file. E.g.:

```scss
/* Allowed */
.foo {
    margin: 5px;
}

.bar {
    padding: 10px;
}

/* Disallowed */
.foo {
    margin: 5px;
}



.bar {
    padding: 10px;
}

```

For JavaScript, we're using ESLint's [no-multiple-empty-lines](http://eslint.org/docs/rules/no-multiple-empty-lines) rule which allows for:

* Max number of allowed consecutive empty lines.
* Separate max configurable for start & end of files.

I'd be happy to try and submit a PR / test cases for same, but wanted to first check if it sounded like a feature others would want to.

Please let me know if I can give more relevant details.

Thanks!
Dave

**Comment by DanPurdy** [comment]
Hi sorry for the delay in getting back to you, had no free time it seems recently. I'd be happy to see a PR submitted for this. Thanks!

**Comment by fiso** [comment]
I would love this too.

**Comment by another-dave** [comment]
Cool, great! I'll try to look into this this week & see if I can get something raised for it.

**Comment by alfaproject** [comment]
I would love this as well. (:

**Comment by fiso** [comment]
This still doesn't exist?

**Comment by fiso** [comment]
For anyone else who feel like they need this rule, I now have a fork where this rule is implemented: https://github.com/fiso/sass-lint

You can try it out by changing your dependency import line to say `"sass-lint": "fiso/sass-lint#develop"` and then running `npm install`.

I'd be happy to submit a PR for it but I feel like I'd want some eyes on my take before I do that, seeing as how I'm completely new to this codebase and all.

**Comment by fiso** [comment]
FWIW my team has been using the forked version for a couple of weeks now and encountered no problems with it.

**Comment by Potherca** [comment]
There seems to be some overlap between this suggested rule and the one suggested in https://github.com/sasstools/sass-lint/issues/1073.

**Comment by fiso** [comment]
FWIW the rule I implemented in my fork (linked above) supports the suggestion from #1073 as well (optionally).

**Comment by Potherca** [comment]
@fiso Awesome! So.. Do you think we should consider #1093 and #1073 as duplicates of this issue?

Also... it might be high time to open that PR! 👍

**Comment by fiso** [comment]
@Potherca Yes that would make sense.

As for the pull request, yes, I've been meaning to do that since forever. I keep putting it off since the checklist for submitting PRs to the project seems kind of daunting. But I'll make time for it.

**Comment by fiso** [comment]
OK, there it is. I've probably made some unintentional faux-pas in the PR. Let me know what needs changing and I'll update it as needed.
